config/default/common/config/metadata/stories/hurricane_dorian_september_2019/step001.md

```diff
@@ -0,0 +1,3 @@
+This true color corrected reflectance image shows Hurricane Dorian over the Bahamas, taken at approximately 18:35 UTC on 1 September 2019. The image was captured by the Visible Infrared Imaging Radiometer Suite (VIIRS) instrument on the joint NASA/NOAA Suomi-National Polar-orbiting Partnership (Suomi-NPP) satellite.
```

Comment:
We capitalize True Color Corrected Reflectance elsewhere in the app, so may want to here/throughout the story.